src/pages/index.js

```diff
@@ -22,207 +21,213 @@ import aboutBlob from "./../../static/images/aboutBlob.png"
-        buttonText: "Browse candidates",
-        image: orange,
-      },
-      {
-        title: "Get the finance facts",
-        description: "Learn more about campaign finance data.",
-        buttonText: "Visit FAQs",
-        image: green,
-      },
-    ],
-    renderItem: BehindTheScenesItem,
-  }
-
-  about = {
-    title: "Power to the people",
-    description:
-      "Open Disclosure was created to empower San José voters with timely, accurate, and useful information about the role of money in local elections.",
-    linkTo: "/",
-    linkImg: learnMore,
-    image: aboutBlob,
-  }
-
-  vote = {
-    title: "Your voice matters",
-    description:
-      "Register to vote or see if you're already registered in less than two minutes.",
-    linkTo: "/registerToVote",
-    linkImg: registerToVote,
-    image: voteBlob,
-  }
-
-  render() {
-    return (
-      <Layout windowIsLarge={this.state.windowIsGreaterThan760px}>
-        <div className={styles.container}>
-          <header className={styles.hero}>
-            <div className={styles.heroLeft}>
-              <h1>
-                More money,
-                <br />
-                more transparency.
-              </h1>
-              <h2>
-                Keep tabs on the influence of money
-                <br />
-                in local San José elections.
-              </h2>
-              <div className={styles.heroButtonContainer}>
-                <Button
-                  secondary
-                  text="View measures"
-                  containerStyle={
-                    this.state.windowIsGreaterThan760px
-                      ? { marginRight: "1.6rem" }
-                      : { marginBottom: "1.6rem" }
-                  }
-                />
-                <Button text="Explore candidates" />
-              </div>
-            </div>
-            <div className={styles.heroRight}>
-              <img
-                alt="header"
-                className="responsive"
-                width="724px"
-                src={headerBlob}
+  return (
+    <Layout windowIsLarge={windowIsLarge}>
+      <div className={styles.container}>
+        <header className={styles.hero}>
+          <div className={styles.heroLeft}>
+            <h1>
+              More money,
+              <br />
+              more transparency.
+            </h1>
+            <h2>
+              Keep tabs on the influence of money
+              <br />
+              in local San José elections.
+            </h2>
+            <div className={styles.heroButtonContainer}>
+              <Button text="Explore candidates" href="/candidates" />
+              <Button
+                secondary
+                text="View measures"
+                href="/measures"
+                containerStyle={
+                  windowIsLarge
+                    ? { marginLeft: "1.6rem" }
+                    : { marginTop: "1.6rem" }
```

Comment:
This looks like it could be done with a media query? (for later)

Reply:
Yes, I removed that in my current branch; I put a div around the first button and styled that rather than pass in styles that didn't work as well when the screen size changed.